### PR TITLE
Watchdog telemetry fixes

### LINKED ISSUE #8049: Update watchdog for new telemetry

### PR DESCRIPTION
**Pull request by es92**
Closes #8049

### CODE REVIEW

**Review by lk86 — APPROVED** [review]

**Comment by es92** [comment]
!approved-for-mainnet

File: automation/services/watchdog/telemetry_metrics.py [hunk 1]
```diff
@@ -121,25 +121,28 @@ def add_resp(resp, direct_queried_peers):
...
-    queried_peers.update([p['peer_id'] for p in direct_queried_peers])
+    queried_peers.update([ p['peer_id'] for p in direct_queried_peers ])
     for p in itertools.chain(*[ p['peers'] for p in peers ]):
-        unqueried_peers[p['peer_id']] = p
+      unqueried_peers[p['peer_id']] = p
     for p in queried_peers:
+      if p in unqueried_peers:
         del unqueried_peers[p]
 
+<<<<<<< HEAD
```
Comment (0x0I) [inline automation/services/watchdog/telemetry_metrics.py:131]: remove (merge-pasta haha)

Comment (es92) [reply automation/services/watchdog/telemetry_metrics.py:131]: Weird, guess python doesn't care? Will remove

**Review by 0x0I — APPROVED** [review]

File: automation/services/watchdog/telemetry_metrics.py (hunk 1 above)
Comment (0x0I) [reply automation/services/watchdog/telemetry_metrics.py]: Whoa really...weird

**Comment by es92** [comment]
!approved-for-mainnet

**Pull request merged**